### PR TITLE
27 create cards routes

### LINKED ISSUE #27: Create cards routes
- [x] update database to accommodate users
    - [x] CardInventory table
    - [x] change cardSearchHSQLDB
    - [x] change CardInventoryHSQLDB
- [x] get Cards
- [x] get a card by its id
- [x] get the cards in the users inventory

### LINKED ISSUE #77: cards query params are not set to correct types
the params for rarities should be setInclude:bool and rarities:int[]

### PR DESCRIPTION
**Pull request by VanJackal**
closes #27 
closes #77

### CODE REVIEW

File: app/src/main/java/com/internetEnemies/combatCritters/data/hsqldb/HSQLDBUserModel.java
Comment (VanJackal) [inline app/src/main/java/com/internetEnemies/combatCritters/data/hsqldb/HSQLDBUserModel.java:1]: cdoc

File: app/src/main/java/com/internetEnemies/combatCritters/objects/CardFilter.java [hunk 1]
```diff
@@ -44,4 +51,16 @@ public Integer getCost() {
     public boolean isLessThan() {
         return lessThan;
     }
+    
+    public User getUser() {
+        return user;
+    }
+    
+    public void clone(ICardFilterBuilder builder){
```
Comment (VanJackal) [inline app/src/main/java/com/internetEnemies/combatCritters/objects/CardFilter.java:59]: missing docstring

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardCritter.java
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardCritter.java:1]: cdoc

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardItem.java
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardItem.java:1]: cdoc

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardPayload.java
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardPayload.java:1]: cdoc

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardPayloadFactory.java
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardPayloadFactory.java:1]: documentation

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardQuery.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        return new CardFilter(
+                rarityInclude != null ? rarityInclude : false, 
+                raritiesList, 
+                owned != null ? owned : false, 
+                cost,
+                costLess!= null ? costLess: false,
+                user);
+    }
+    
+    public CardOrder toOrder(){
```
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/CardQuery.java:60]: docstring

File: critterSpring/src/main/java/com/combatcritters/critterspring/payloads/ItemStackPayload.java
Comment (VanJackal) [inline critterSpring/src/main/java/com/combatcritters/critterspring/payloads/ItemStackPayload.java:1]: cdoc

**Review by kivine99 — APPROVED** [review]
I don't see anything wrong here...

**Pull request merged**